src/transformStateWithClones.js

```diff
@@ -7,7 +7,41 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const results = [];
+  let temporaryPreviousObject = { ...state };
+
+  for (const item of actions) {
+    const type = item.type;
```

Comment:
```suggestion
  for (const { type... } of actions) {
```
You can destructure all the needed props directly

Reply:
Yeah, I forgot to use all JSs features